packages/content-conformance/src/config.ts

```diff
@@ -83,3 +96,32 @@ function applyConfigDefaults(config: ContentConformanceConfig) {
 function resolveConfigRoot(rootValue: string, cwd: string) {
   return path.resolve(cwd, rootValue)
 }
+
+/**
+ * Attempts to load a preset, if `config.preset` is defined, and merge them. The loaded config takes precedence over the preset.
+ */
+async function resolveConfigPreset(
```

Comment:
💡 
How do you feel about logging the fact that a preset was loaded?

(inspired by the nextjs compiler's feedback for env files)

```console
info  - Loaded env from /Users/kevin/repos/express-app/.env.local
info  - Loaded env from /Users/kevin/repos/express-app/.env
```

Additionally (related but out of scope of this PR) how do you feel about logging the fact that a config was loaded as well?

Reply:
sure sure, good idea!